source/read/project.txt

```diff
@@ -0,0 +1,137 @@
+.. _pymongo-project:
+
+========================
+Specify Fields To Return
+========================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: read, filter, project, select
+
+Overview
+--------
+
+In this guide, you can learn how to specify which fields to return from a read
+operation by using a **projection**. A projection is a document supplied to a 
+query that specifies which fields MongoDB returns.
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide use the ``sample_restaurants.restaurants`` collection
+from the :atlas:`Atlas sample datasets </sample-data>`. To learn how to create a
+free MongoDB Atlas cluster and load the sample datasets, see the
+:ref:`<pymongo-get-started>` guide.
+
+Projection Types
+----------------
+
+You can use a projection to specify which fields to include in a return
+document, or to specify which fields to exclude. With the exception of the
+``_id`` field, you can't combine inclusion and exclusion statements in
+a single projection.
+
+Specify Fields to Include
+~~~~~~~~~~~~~~~~~~~~~~~~~
+
+When you use a projection to specify fields to include, any unspecified fields
+are implicitly excluded from the return document. 
+
+.. note::
+    
+   The ``_id`` field is *not* implicitly excluded from the return document when
+   it isn't included in the projection. To remove the ``_id`` field from the return
+   document, you must :ref:`explicitly exclude it <pymongo-project-remove-id>`.
```

Comment:
Possible shorter version of this text:

When you use a projection to specify fields to include in the return
document, the _id field is also included by default. All other fields are
implicitly excluded.